sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -288,6 +288,15 @@ public static void setLogUrl(String logUrl) {
         getInstance().setInstanceLogUrl(logUrl);
     }
 
+    /**
+     * Set the two-letter ISO country code.
```

Comment:
Why don' we use the same comment as in [Apple SDK](https://github.com/microsoft/appcenter-sdk-apple/blob/develop/AppCenter/AppCenter/MSACAppCenter.h#L182)?

Reply:
this comment from dotnet